src/seq_tensor.cpp

```diff
@@ -208,10 +208,8 @@ SeqTensorDescriptor::SeqTensorDescriptor(miopenDataType_t t,
     else
     {
         if(padding_in.size() != dims)
-        {
```

Comment:
@atamazov well it should never hurt to have an extra pair of {}  ;-P

Reply:
@junliume Yes except two things:
- These may give a false impression that we need braces because of the macro (which is not so)
- They waste screen space (which I personally value very high even with my 2560x1600 display).

But I do not insist on these things (all the more they eventually sneak into our code). Just let me know and I'll bring'em back.